client/src/pages/PaymentError/PaymentError.scss

```diff
@@ -0,0 +1,129 @@
+@import '../../scssStyles/mixins';
+
+.payment-error-page {
+  width: 100vw;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.payment-error-container {
+  display: flex;
+  flex-direction: row;
+  padding: 20px;
+  justify-content: center;
+  align-items: center;
+
+  border-radius: 20px;
+  background: white;
+  box-shadow: var(--shadow);
+
+  margin: 20px;
+
+  @include devices(tablet) {
+    flex-direction: column;
+  }
+}
+
+.payment-error-image {
+  flex: 1;
+  width: 30%;
+  animation: error-image-float-up-down 3s infinite alternate-reverse;
+
+  @include devices(mobile) {
+    width: 50%;
+  }
+}
+
+@keyframes error-image-float-up-down {
+  0% {
+    transform: translateY(0);
+  }
+  80% {
+    transform: translateY(20px);
+  }
+  100% {
+    transform: translateY(20px);
+  }
+}
+
+.payment-error-text-container {
+  flex: 1;
+  padding-left: 80px;
+
+  @include devices(tablet) {
+    padding: 5px;
+    text-align: center;
+    justify-content: center;
+    align-items: center;
+  }
+}
+
+.payment-error-text-container-title {
+  font-size: 60px;
+  font-weight: bold;
+  padding: 10px 10px 25px 10px;
+  color: var(--purple);
+  width: fit-content;
+
+  transition: 500ms ease-in-out all;
```

Comment:
All shouldn't be used as it reduces the performance drastically 
See `Avoiding animation pollution` in
https://github.blog/2021-01-29-making-githubs-new-homepage-fast-and-performant/

Reply:
Seems like this transition isn't needed actually, but you can still take a look at that document if you wish